src/dotnet/APIView/APIView/Analysis/Analyzer.cs

```diff
@@ -20,7 +20,6 @@ public Analyzer()
             _analyzers.Add(new ClientMethodsAnalyzer());
             _analyzers.Add(new ClientConstructorAnalyzer());
             _analyzers.Add(new ClientOptionsAnalyzer());
-            _analyzers.Add(new ClientAssemblyNamespaceAnalyzer());

```

Comment:
Can you please rebase to remove these changes from other PR if it's already merged?